packages/cli/src/lib/graphql.test.ts

```diff
@@ -1,18 +0,0 @@
-import {describe, it, expect} from 'vitest';
-import {AbortError} from '@shopify/cli-kit/node/error';
-
-import {parseGid} from './graphql.js';
-
-describe('parseGid', () => {
```

Comment:
Do we have tests for the remaining implementation?

Reply:
Yup! https://github.com/Shopify/hydrogen/blob/c1937518a70b2adec741792306a667946af09d74/packages/hydrogen-react/src/analytics-utils.test.ts#L5